test/test_art.py

```diff
@@ -176,38 +192,46 @@ def test_main_interface_returns_none_for_missing_asin_and_path(self):
     def test_main_interface_gives_precedence_to_fs_art(self):
         _common.touch(os.path.join(self.dpath, b'art.jpg'))
         self.mock_response(self.AMAZON_URL)
+        self.mock_response(self.AMAZON_URL_HTTPS)
         album = _common.Bag(asin=self.ASIN)
         candidate = self.plugin.art_for_album(album, [self.dpath])
         self.assertIsNotNone(candidate)
         self.assertEqual(candidate.path, os.path.join(self.dpath, b'art.jpg'))
 
     def test_main_interface_falls_back_to_amazon(self):
         self.mock_response(self.AMAZON_URL)
+        self.mock_response(self.AMAZON_URL_HTTPS)
         album = _common.Bag(asin=self.ASIN)
         candidate = self.plugin.art_for_album(album, [self.dpath])
         self.assertIsNotNone(candidate)
         self.assertFalse(candidate.path.startswith(self.dpath))
 
     def test_main_interface_tries_amazon_before_aao(self):
         self.mock_response(self.AMAZON_URL)
+        self.mock_response(self.AMAZON_URL_HTTPS)
         album = _common.Bag(asin=self.ASIN)
         self.plugin.art_for_album(album, [self.dpath])
         self.assertEqual(len(responses.calls), 1)
         self.assertEqual(responses.calls[0].request.url, self.AMAZON_URL)
+        self.assertEqual(responses.calls[0].request.url, self.AMAZON_URL_HTTPS)
```

Comment:
This seems to assert that the requested URL is equal to two *different* URLs, which probably isn't what you meant—instead, you probably want to write an assertion that says that the requested URL is *either* the HTTP one or the HTTPS one.